src/datumaro/components/hl_ops/__init__.py

```diff
@@ -35,6 +36,7 @@
 __all__ = ["HLOps"]
 
 
+@deprecated()
```

Comment:
Let’s keep this class as it has useful operations like filter and export; though, we may refactor it in the future. We can deprecate the run_model function.